CMakeLists.txt

```diff
@@ -22,8 +21,6 @@ add_executable(${APP_TARGET})
 
 mbed_configure_app_target(${APP_TARGET})
 
```

Comment:
@harmut01 Sorry I missed this earlier - this can be removed too, though it makes no difference to have it.